woeip/apps/air_quality/tests/test_factories.py

```diff
@@ -0,0 +1,13 @@
+import pytest
```

Comment:
You don't need to test factories. Your tests will do that for you.

Reply:
Yeah I figured that there was this bug in the code (``DataFactory`` didn't work), so I'd retroactively put a test in there that would have caught that bug. I agree the better solution would be to have tests that used the factories in a more meaningful way, but for now I'd like to keep something in there that would catch any future bugs in the factories.

Reply:
If there is a bug in a factory, it will result in failed tests that use said factory. Retaining this test doesn’t do anything your other tests aren’t already doing.

Reply:
This test does do something our other tests aren't doing, namely test ``DataFactory``. I guess I could write a single test for ``DataFactory`` until we have more comprehensive tests.

Reply:
There is a simpler solution: remove DataFactory. If you don’t have tests that need the factory, you don’t need the factory. Remove it until you need it.

Reply:
Ha, that's a good idea.